01_prepare_host.sh

```diff
@@ -33,6 +33,11 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
   -i vm-setup/inventory.ini \
   -b -vvv vm-setup/install-package-playbook.yml
 
+# Add usr/local/go/bin to the PATH environment variable
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc
+# shellcheck disable=SC1090
+source ~/.bashrc
+
```

Comment:
you'll end up with as many lines like this as you run this script. Please first check that the line is not present before adding it. Also,there's no need to source the file here, considering that the variables are already properly set when running this script. So only line 37 (with a condition) would be needed

Reply:
Good catch, should be fixed now. Thanks